### ISSUE TITLE
Building RTKLIB in OS X 10.10.5

### ISSUE THREAD

**Issue by chenosaurus** [tomojitakasu/RTKLIB#137]
Trying to compile the CLI tools on Mac OS X and running into some issues.

I've removed the '-lrt' from the makefile in str2str, but now the error is different:

cc   str2str.o stream.o rtkcmn.o solution.o sbas.o geoid.o rcvraw.o novatel.o ublox.o ss2.o crescent.o skytraq.o gw10.o javad.o nvs.o binex.o rt17.o rtcm.o rtcm2.o rtcm3.o rtcm3e.o preceph.o streamsvr.o  -lm -lpthread -march=i686 -o str2str
Undefined symbols for architecture x86_64:
  "_input_sbf", referenced from:
      _input_raw in rcvraw.o
  "_input_sbff", referenced from:
      _input_rawf in rcvraw.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [str2str] Error 1

Anyone been able to compile this on Mac?

**Comment by JensReimann** [comment]
I looks like the file src/rcv/septentrio.c is missing in the makefile.

**Comment by blaines** [comment]
@chenosaurus I updated the makefiles and pushed to my branch. Enjoy!

**Comment by ljbade** [comment]
@blaines Thanks this fixes Linux too.

**Comment by coandrei** [comment]
@chenosaurus -lrt (POSIX real time extensions) is not available for Mac. Indeed one solution would be to remove it completely. Although I am not sure how the complete removal affects the behaviour of the executable.
On the other hand, according to this [link](http://lists.apple.com/archives/unix-porting/2008/Oct/msg00001.html), the real time extensions would be located in libSystem.B.dylib. As a result, you may replace _-lrt_ with _-lSystem.B_

**Comment by demiangomez** [comment]
I noticed that this issue has been closed, but maybe someone can help me. I've successfully compiled in OS X 10.12.3 (Sierra) using -lSystem.B as suggested by @coandrei. However, when I try to run rnx2rtkp I get "Segmentation fault: 11". I also tried removing -lrt (without adding -lSystem.B) and I get the same problem. In Linux (Ubuntu 14.04) works fine (without changing -lrt). Anyone has any ideas of how to solve this in Sierra?
